### ISSUE TITLE
TypeError: Cannot read property 'startedAt' of undefined

### ISSUE THREAD

**Issue by BrunoLagoa** [infinitered/reactotron#917]
Hello,
I have this error below, can you help me? I do not know what to do.

> TypeError: Cannot read property 'startedAt' of undefined
> updateDuration

```
var updateDuration = function updateDuration(effectInfo) {
> 142 |   effectInfo.duration = timer() - effectInfo.startedAt;
      | ^  143 | }; // fires when a task has been resolved


var effectResolved = function effectResolved(effectId, result) {
  241 |   // lookup this effect info and set the rsult
  242 |   var effectInfo = effects[effectId];
> 243 |   updateDuration(effectInfo);
      | ^  244 |   effectInfo.result = result; // this is a task
  245 | 
  246 |   if (reduxSaga_utils.is.task(result)) {



  1369 | true, noop);
  1370 | 
  1371 | if (sagaMonitor) {
> 1372 |   sagaMonitor.effectResolved(effectId, task);
       | ^  1373 | }
  1374 | 
  1375 | return task;
```

**Comment by ahmad2smile** [comment]
Same problem. Just updated `"reactotron-redux-saga": "^2.1.3"` to `"reactotron-redux-saga": "^2.1.4"`

**Comment by ahmad2smile** [comment]
Did little bit of debugging `effects` for `effectId = 1` is `undefined`.

**Comment by rostislav-simonik** [comment]
@ahmad2smile yes it's related to redux-saga@1.0.0. The issue is already addressed in pending PR. https://github.com/infinitered/reactotron-redux-saga/pull/5

**Comment by rmevans9** [comment]
Version 3.0.0 of reactotron-redux-saga has been released that supports redux-saga 1.0.0 🎉
